script/DeployUniswapV2Pool.s.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.28;
+
+import "forge-std/Script.sol";
+import {Utils} from "script/libraries/Utils.sol";
+import {EnvReader} from "script/libraries/EnvReader.sol";
+
+interface IUniswapV2Factory {
```

Comment:
I don't think this should be part of the repository. If the backend needs this for testing, we should set this up one-off but not check it in as this repo has nothing to do per se with Uniswap v2 pools.

Reply:
Makes sense. Will remove.